countdown_timer.jsx

```diff
@@ -126,7 +126,7 @@ var CountdownTimer = React.createClass({
     var timeRemaining = this.state.timeRemaining;
 
     return (
-      <div className='timer'>
+      <div className={this.props.setClass || 'timer'}>
```

Comment:
For simplicity, I think this component should expect a string. That way we don't need to add any type-checking or conversion logic to this component (I think that's out of scope given what it does). If a dev needs to add multiple classes, they can simply pass in a pre-formatted string with spaces between the class names they'd like to add.